app/controllers/users_controller.rb

```diff
@@ -21,10 +28,55 @@ def create
     end
   end
 
+  def edit
+    @user = User.find_by id: params[:id]
+  end
+
+  def update
+    @user = User.find_by id: params[:id]
+    if @user.update_attributes(user_params)
+      flash[:sucess] = t ".profile_update"
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @user = User.find params[:id]
```

Comment:
You must control exception when find method don't find record in your system.

Reply:
fixed